### PR TITLE
Powergate FAQs

### PR DESCRIPTION
**Pull request by jsign**
Powergate FAQs gathered by some days of doing support in #powergate-users.

I'm not completely sure if using `code quotes` in ## sections is going to be rendered nicely. 
Maybe there is a better-known strategy?

Anything extra apart from editing `mkdocs` for including this new file?

### CODE REVIEW

File: docs/powergate/faq.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Filecoin Testnet
+
+Here are some questions that usually get asked by the community.
+
+### Which Powergate CLIs commands are a minimal test for using localnet?
```
Comment (andrewxhill) [inline docs/powergate/faq.md:5]: maybe: 'What is a minimal test of localnet using the Powergate CLI?'

File: docs/powergate/faq.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+> 2020-09-28T09:12:16 - Estimated piece size is 16777216 bytes.
+> 2020-09-28T09:12:16 - Proposing deal to miner t01000 with 500000000 attoFIL per epoch...
+> 2020-09-28T09:12:17 - Watching deals unfold...
+> 2020-09-28T09:12:47 - Deal with miner t01000 changed state to StorageDealSealing
+> 2020-09-28T09:14:47 - Deal 2 with miner t01000 is active on-chain
+> 2020-09-28T09:14:47 - Cold-Storage execution ran successfully.
+> 2020-09-28T09:14:47 - Job e27f1366-f3d0-484d-a0b3-b3abe71c150c execution finished with status Success.
+```
+
+### Why my custom StorageConfig seems to be ignored?
```
Comment (andrewxhill) [inline docs/powergate/faq.md:31]: 'Why is my custom StorageConfig being ignored?'

File: docs/powergate/faq.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+
+### What does the following log error mean:  `rpc go-jsonrpc: xxxx`?
+
+Usually, this is related to a [reported](https://github.com/filecoin-project/lotus/issues/3581) issue. As a solution, Powergate switched to another style of connecting to Lotus which is more reliable and avoids that problem to get in the way. 
+You could ignore those errors since Powergate is not relying on the Lotus client to have stable connections now. Whenever you see those errors not impacting other logs related to powergate screaming other business errors, you should be fine; if that isn't the case please report in a support channel.
+
+We preferred not to completely hide errors this external package just in case we discover new problems.
+
+### I've made a deal in Testnet/SR2, how can I know if all is doing well?
```
Comment (andrewxhill) [inline docs/powergate/faq.md:82]: `I've made a deal in Testnet/SR2, how can I know everything is working okay?`

File: docs/powergate/faq.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+We recommend using the `pow ffs -t <ffs-token> log <pushed-cid>` which provides a human-friendy output.
+Creating deals in testnet can take more than 10hs in the usual case, and many things can go wrong since Powergate is being relatively open to miners that provide storage in the network, so they might go offline at any time, be unreliable, or have network problems.
```
Comment (andrewxhill) [inline docs/powergate/faq.md:85]: `hs` => `hrs` more commonly

**Review by andrewxhill — APPROVED** [review]

**Comment by jsign** [comment]
@andrew , thanks for the review... I wrote this thing too fast.

**Pull request merged**